bin/helpers/utils.js

```diff
@@ -327,6 +327,39 @@ exports.setSpecTimeout = (bsConfig, args) => {
   bsConfig.run_settings.spec_timeout = specTimeout;
 }
 
+exports.setRecordFlag = (bsConfig, args) => {
+  if(!this.isUndefined(args["record"])) {
+    return true;
+  }
+  return bsConfig.run_settings["record"];
+}
+
+exports.setRecordKeyFlag = (bsConfig, args) => {
+  if(!this.isUndefined(args["record-key"])) {
+    return args["record-key"];
+  } else if (!this.isUndefined(process.env.CYPRESS_RECORD_KEY)) {
+    return process.env.CYPRESS_RECORD_KEY;
+  }
+  return bsConfig.run_settings["record-key"];
+}
+
+exports.setProjectId = (bsConfig, args) => {
+  if(!this.isUndefined(args["projectId"])) {
+    return args["projectId"];
+  } else if(!this.isUndefined(bsConfig.run_settings["projectId"])) {
```

Comment:
Just like env var `CYPRESS_RECORD_KEY`, we have an env var for project id i.e. `CYPRESS_PROJECT_ID`. More info is available [here](https://docs.cypress.io/guides/dashboard/projects#Project-ID).